packages/open-payments/src/client/requests.ts

```diff
@@ -65,12 +65,36 @@ export const get = async <T>(
 
 export const createAxiosInstance = (args: {
   requestTimeoutMs: number
+  privateKey: KeyLike
+  keyId: string
 }): AxiosInstance => {
   const axiosInstance = axios.create({
     timeout: args.requestTimeoutMs
   })
-
   axiosInstance.defaults.headers.common['Content-Type'] = 'application/json'
 
+  axiosInstance.interceptors.request.use(
```

Comment:
Confirming: does this apply to all GET, POST, DELETE requests?

Reply:
I believe so, but it can be filtered.